rollup.config.mjs

```diff
@@ -25,10 +25,10 @@ import typescript from 'rollup-plugin-typescript2';
 
 // generate warning when running with Node 18
 // (node:75278) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time
-import pkg from './package.json' assert { type: 'json' };
+import package_ from './package.json' assert { type: 'json' };
```

Comment:
**suggestion:** rename into `packageJson` for clarity (the `_` postfix is weird).